src/main/java/virtual_pet/VirtualPetApplication.java

```diff
@@ -1,9 +1,100 @@
+                System.out.println("To do nothing at this time, type 5");
+                System.out.println("To end the game, type 6");
+
+                Scanner scanner1 = new Scanner(System.in);
+                int userSelection = Integer.valueOf(scanner1.next());
+
+                 System.out.println("Current hunger level is " + Rick.getHungerLevel());
+            System.out.println("Current thirst level is " + Rick.getThirstLevel());
+            System.out.println("Current boredom level is " + Rick.getBoredomLevel());
+            System.out.println("Current need for a bathroom break is " + Rick.getBathroomBreak());
+
+                if (userSelection == 1) {
+                    Rick.feedPet();
+                } else if (userSelection == 2) {
+                    Rick.givePetWater();
+                } else if (userSelection == 3) {
+                    Rick.playWithPet();
+                } else if (userSelection == 4) {
+                    Rick.givePetBathroomBreak();
+                } else if (userSelection == 5) {
+//                    Rick.tick();
+                    System.out.println("Time is ticking!");
+                } else if (userSelection == 6) {
+                    System.out.println("Thank you for playing with my pet!");
+                    System.exit(0);
+                } else if (userSelection == 0) {
+                    printUserInstructions();
+                }
+
+                if (userSelection != 0) {
+//                    Rick.tick();
+                    displayCurrentNeeds();
+                }
+                displayCurrentNeeds();
+
+
+                System.out.println("What would you like to do now? (type 0 for user instructions)");
+            }
+            System.out.println("Pet ran away !");
+        }
+
+         {
+
+            System.out.println("Objective: try to keep the pet happy so he doesn't run away!");
+            displayCurrentNeeds();
+            printUserInstructions();
+        }
+        {
+
+            System.out.println("Current hunger level is " + Rick.getHungerLevel());
+            System.out.println("Current thirst level is " + Rick.getThirstLevel());
+            System.out.println("Current boredom level is " + Rick.getBoredomLevel());
+            System.out.println("Current need for a bathroom break is " + Rick.getBathroomBreak());
+        }
+
+
+    private static void printUserInstructions() {
+
+        System.out.println("What would you like to do? ");
+        System.out.println("Want to feed, type 1");
+        System.out.println("Want to water, type 2");
+        System.out.println("To play with pet, type 3");
+        System.out.println("To give a bathroom break, type 4");
+        System.out.println("To do nothing at this time, type 5");
+        System.out.println("To end the game, type 6");
+
     }
 
+    private static void displayGameObjective() {
+    }
+
+    private static boolean areNeedsTooHigh() {
```

Comment:
Method function not used.  Should be removed from your code.  Furthermore, why create a method function for this when you could just declare a primitive variable as boolean.